addon/components/infinity-loader.js

```diff
@@ -13,6 +13,7 @@ export default Ember.Component.extend({
   destroyOnInfinity: false,
   developmentMode: false,
   scrollable: null,
+  didSendActionFromScrollEvent: false,
```

Comment:
It's used to prevent sending the route an action twice. Are you suggesting sending the route both actions and have the route decide not to act on one? In that case, the logic to see if the loader is in view in `_checkIfInView` will unnecessarily run a second time.